geograypher/meshes/meshes.py

```diff
@@ -123,6 +123,16 @@ def __init__(
         )
         # Load the texture
         self.logger.info("Loading texture")
+        # load IDs_to_labels
+        # if IDs_to_labels not provided, check the directory of the mesh and get the file if found
+        if IDs_to_labels == None and isinstance(mesh, PATH_TYPE.__args__):
+            for file in mesh.parent.iterdir():
```

Comment:
Rather than iterating over files, can you just check if the filename you expect exists? I think this would be cleaner code and more robust. One edge case that comes to mind is if the user saves two meshes, each with their own sidecar, to the same folder.